### ISSUE TITLE
EfficientB0 on cifar10  without transfer learning and trainng from scratch

### ISSUE THREAD

**Issue by ww-zwj** [lukemelas/EfficientNet-PyTorch#204]
If I trained the efficient netb0 from scratch and input size is 32*32, what is accuracy the network now on the cifar-10 ?

**Comment by lukemelas** [comment]
I haven't tested it. Networks for CIFAR are usually somewhat different from networks for ImageNet (due to image size and statistics). You're probably better off using a WideResNet or something like that.

**Comment by siarez** [comment]
@ww-zwj I'm trying this now with Cifar100. B0 and B5 both get around %60 test accuracy which is much worse than ResNet18 or ResNet50. 
Did you figure out the right hyper-parameters?
